jetstream/cli.py

```diff
@@ -706,6 +706,10 @@ def validate_config(path: Iterable[os.PathLike]):
             print(f"Skipping example config {config_file}")
             continue
         print(f"Evaluating {config_file}...")
+
+        external_config.validate_config_settings(config_file)  # see Github issue for more context:
+        # https://github.com/mozilla/jetstream/issues/843
+
         entity = external_config.entity_from_path(config_file)
```

Comment:
I think we can move the `external_config.validate_config_settings(config_file)` call into  the `external_config.entity_from_path` method. Just in case we call `entity_from_path` somewhere else, we'd still want to do the validation.

I think we also want to call `validate_config_settings` in [`ExternalConfig.from_github_repo()`](https://github.com/mozilla/jetstream/blob/6c8d56811cacb25263beac3f9a24dafc37c4287d/jetstream/external_config.py#L103). In theory it is possible to skip the CI validation and check in invalid configs. The problem here (I believe) was that no error would be thrown, instead metrics with typos just get ignored. But it would be nice actually throw an error so that experiment owners know that something is off and what they need to fix.

Reply:
Moved it and added to `from_github_repo()` too.